### ISSUE TITLE
Global Color Selection

### ISSUE THREAD

**Issue by FriederHannenheim** [flxzt/rnote#154]
I think that a global color selection would be better than having different color selectors for pens and shapes the way it is now. It could be in the top bar to the right of the document name.
I was taking notes in class and wanted to make a straight arrow so I created a line with the shapes tool and then wanted to draw the arrow head using the brush but the color was different. This is a bit confusing and should be resolved.

**Comment by flxzt** [comment]
Maybe this is not such a bad idea. It is often annoying that one has to change the color separately for the shapes.
I just don't know where to place a global color selection yet.

**Comment by mahdi4261** [comment]
Well it does cause other issues. I mostly use shaper tool to mark something such as math results. So I use differnt color for that. Now I need to change to shaper tool, change color, mark, change to pen tool and change color again. So, I changed the source code and rebuilt the app to get independent color for each tool. But, now I discovered it was default once upon a time. 😅

<img width="1343" height="1030" alt="Image" src="https://github.com/user-attachments/assets/1069a3f2-cb86-4bd6-9f1d-7e78d6dabc19" />
